packages/server/src/migrations/index.ts

```diff
@@ -1,4 +1,4 @@
 import { Migration } from '@botpress/messaging-engine'
-import { StatusMigration } from './0.1.18-status'
+import { StatusMigration } from './0.1.19-status'
```

Comment:
This migration had the wrong file name, but it has the correct server version in it's metadata so it shouldn't have caused problems